### ISSUE TITLE
Failing *:_isolated/* tests are (partially) double-counted in summary

### ISSUE THREAD

**Issue by jamiesnape** [RobotLocomotion/drake#8181]
I guess this is a Bazel bug/feature of sorts, but I will document here for triage for now.
```
INFO: Build completed, 1 test FAILED, 3277 total actions
//tools/workspace:_isolated/bazel_codestyle                              FAILED in 0.7s
  /private/var/tmp/_bazel_user/7c7ad393d9017d670ea6cd4ae3566d69/execroot/drake/bazel-out/darwin-opt/testlogs/tools/workspace/_isolated/bazel_codestyle/test.log
//tools/workspace:bazel_codestyle                                        FAILED in 0.7s
  /private/var/tmp/_bazel_user/7c7ad393d9017d670ea6cd4ae3566d69/execroot/drake/bazel-out/darwin-opt/testlogs/tools/workspace/_isolated/bazel_codestyle/test.log
```

**Comment by EricCousineau-TRI** [comment]
Huh, odd... Thanks for the heads up!

**Comment by jwnimmer-tri** [comment]
Fixed in ff3f48990f1b76fa172d16431d7548721844811e.
